doc/releases/changelog-dev.md

```diff
@@ -177,7 +177,22 @@
   interface to maintain a list of special implementations.
   [(#7327)](https://github.com/PennyLaneAI/pennylane/pull/7327)
 
-* Sphinx version was updated to 8.1. Sphinx is upgraded to version 8.1 and uses Python 3.10. References to intersphinx (e.g. `<demos/>` or `<catalyst/>` are updated to remove the :doc: prefix that is incompatible with sphinx 8.1. [(7212)](https://github.com/PennyLaneAI/pennylane/pull/7212)
+* Sphinx version was updated to 8.1. Sphinx is upgraded to version 8.1 and uses Python 3.10.
+  References to intersphinx (e.g. `<demos/>` or `<catalyst/>` are updated to remove the
+  :doc: prefix that is incompatible with sphinx 8.1.
+  [(7212)](https://github.com/PennyLaneAI/pennylane/pull/7212)
+
+<h3>Labs: a place for unified and rapid prototyping of research software 🧪</h3>
+
+* A `rowcol` function is now available in `pennylane.labs.intermediate_reps`.
```

Comment:
changelog needs some cleaning but I guess that is due to merge conflicts after I merged the parent branch to main

Reply:
I took the liberty to just push this 😅

Reply:
Oh, thanks! :pray: